integration/tests/cook/test_multi_user.py

```diff
@@ -114,6 +114,8 @@ def test_multi_user_usage(self):
                 with self.user_factory.admin():
                     util.kill_jobs(self.cook_url, all_job_uuids, assert_response=False)
 
+    # Serial so that we don't conflict with another running test that may use the same user ID.
+    @pytest.mark.serial
```

Comment:
This is already guaranteed by the `UserFactory`

Reply:
Ok. Removed.